docs/_coverpage.md

```diff
@@ -2,7 +2,7 @@
  <img src="static/logo.svg" alt="kubevela logo" width=300 height=300 />
 </p>
 
-> Make shipping applications more enjoyable.
+> An Easy-to-use yet Highly Extensible App Platform based on Kubernetes and Open Application Model.
```

Comment:
Keep the original one please as it's the slogan of KubeVela.

Reply:
ok